intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -423,6 +423,73 @@ In this workshop, we will not delve too deeply into the various other apply
 functions (`tapply()`, `lapply()`, etc.) but you can read more information about 
 them [here](https://www.guru99.com/r-apply-sapply-tapply.html).
 
+## Matrices
+
+A matrix might at first appear similar to a data frame, but it has an additional
+constraint: rather than each column having a single type, ALL data in the matrix
+has the same type.
```

Comment:
```suggestion
A matrix is similar to a data frame in that it is a rectangular table of data, but it has an additional constraint: 
rather than each column having a type, ALL data in a matrix has the same type.
```